lib/index.js

```diff
@@ -0,0 +1,6 @@
+var foodsRequests = require('./requests/foods')
+require("./foods")
+
+$(document).ready(() => {
```

Comment:
This works for now but will have to investigate how to separate out according to which file/document functionality is related to.